src/pyOpenMS/pxds/DigestionEnzyme.pxd

```diff
@@ -7,21 +7,35 @@ from String cimport *
 cdef extern from "<OpenMS/CHEMISTRY/DigestionEnzyme.h>" namespace "OpenMS":
     
     cdef cppclass DigestionEnzyme "OpenMS::DigestionEnzyme":
-        DigestionEnzyme(DigestionEnzyme) nogil except + # TODO
-        DigestionEnzyme(const String & name, const String & cleavage_regex, libcpp_set[ String ] & synonyms, String regex_description) nogil except + # TODO
+        # wrap-doc:
+        #     Abstract base class for digestion enzymes
```

Comment:
not sure but I think DigestionEnzyme is only a base class not an abstract base class.

Reply:
hm, see https://github.com/OpenMS/OpenMS/blob/develop/src/openms/include/OpenMS/CHEMISTRY/DigestionEnzyme.h#L50

Reply:
But I think you are right since DigestionEnzyme can be instantiated, so it is not an abstract base class, or?

Reply:
yep I think so. Can you change the comment to "base class"